past-issues/src/main/res/values/ids.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <item name="fragment_section_details" type="id" />
+    <item name="item_menu_activity_main_latest_issue" type="id" />
```

Comment:
What is this for?

Reply:
For call latest issue fragment's id in navigate function.